0-bootstrap/main.tf

```diff
@@ -29,11 +29,17 @@ resource "google_folder" "bootstrap" {
   parent       = local.parent
 }
 
+resource "random_id" "suffix" {
+  byte_length = 2
+}
+
 module "seed_bootstrap" {
   source                         = "terraform-google-modules/bootstrap/google"
   version                        = "~> 2.1"
   org_id                         = var.org_id
   folder_id                      = google_folder.bootstrap.id
+  project_id                     = "prj-example"
+  state_bucket_name              = "bkt-b-tfstate-${random_id.suffix.hex}"
```

Comment:
@bharathkkb - we need to add the random suffix here to ensure builds don't fail / good UX if users don't change defaults. Do you think it might be worth adding a flag to bootstrap module `random_suffix` to toggle if users want that for project / bucket and other resources? (we could do this separately and then remove this later)

Reply:
@rjerrems that sgtm as a follow up

Reply:
@rjerrems  Considering the naming examples in the guide and the reason we added the `var.project_prefix` to prevent namespace collision, shouldn't the bucket be named as:

```terraform
"bkt-${project_prefix}-b-seed-tfstate-${random_id.suffix.hex}"
```

Reply:
The name that would generate would not be the best, it would expand to something like "bkt-prj-b-seed-tfstate-237sd" if the user does not change the defaults, but I suppose it is closer to the guide with the `bkt` prefix at the beginning

Reply:
In the guide the description is:

```
bkt-project-name{-bucket-label}
```
and the example is

```
bkt-p-acde-shared-base-data
```

one problem we can have if the prefix is not used is that the buckets from different projects would fall in to the same similar patterrn:

`prj-b-seed-<random_1>`  **=>** `bkt-b-seed-tfstate-<random_2>`
and 
`tst-b-seed-<random_3>`  **=>**  `bkt-b-seed-tfstate-<random_4>`

since there is only one namespace for bucket names it would increase the chance of a collision.